### DISCUSSION TITLE
claimslist question

### DISCUSSION THREAD

**Discussion by mjmeans** [GriefPrevention/GriefPrevention#1358] [General]
So claimslist reports one corner and the total number of blocks. Is there a command that will instead display either opposite corners, or an origin corner plus separate x & z lengths? The blocks total used in a claim is of no interest to me as I can do the simple math if I had either both opposite corners, or one corner and x/z lengths. However, the x/z lengths of each claim are, to me, more important than total blocks.

**Comment by bigpresh** [comment]
This is a fair point I think.  I don't think there is a separate command which would get you this info, but the claimslist output could be more helpful.

Rather than e.g.:

`world: x12345, z5678 (-375 blocks)`

it could tell you the dimensions as well as blocks used, e.g.:

`world: x12345, z5678 (25x15 = 375 blocks)`

... or similar.

**Comment by RoboMWM** [comment]
yea, the basic claimslist could be improved upon, probably will do so in v20 or so. However in its basic form it's more or less used for players to see how many claimblocks are in use for their claims.

**Reply by mjmeans** [reply]
With more information it would be easier to subdivide a claim based on size or parcel out lots from a large claim. I would suggest though that the information be an ordered dimensioned set of bounds such as: "world: (x12345, z5678):(x12360, z5673) = (375 blocks).
